File: questions/Day1/Answers/q2.c
// 2) Create a program to print the values of a = 10,b=25.42,c="cat",d=TRUE.

#include <stdio.h>
#include <stdbool.h>
int main(){
    int a = 10;
    float b = 25.42;
    char c[]="cat";
    bool d = true;
    printf("a is %d\n",a);
    printf("b is %f\n",b);
    printf("c is %s\n",c);
    printf("d is %d\n",d);
    return 0;
}

File: questions/Day1/Answers/q2.py
# 2) Create a program to print the values of a = 10,b=25.42,c="cat",d=TRUE.

# simple answer---------------------------------------------------------
a = 10
b = 25.42
c = "cat"
d = True

print(f"a = {a}")
print(f"b = {b}")
print(f"c = {c}")
print(f"d = {d}")


# Medium answer---------------------------------------------------------
a = 10
b = 25.42
c = "cat"
d = True

def printVals():
    print(f"a = {a}")
    print(f"b = {b}")
    print(f"c = {c}")
    print(f"d = {d}")

printVals()


# Hard answer---------------------------------------------------------
a = 10
b = 25.42
c = "cat"
d = True

def printVals(a,b,c,d):
    print(f"a = {a}")
    print(f"b = {b}")
    print(f"c = {c}")
    print(f"d = {d}")

printVals(a,b,c,d)
